internal/pkg/scorecard/scorecard.go

```diff
@@ -49,54 +51,69 @@ var (
 	log           = logrus.New()
 )
 
-var rootDir string
+var scViper *viper.Viper
 
-func getPlugins() []Plugin {
-	var plugins []Plugin
-	// Add internal plugins
-	if viper.GetBool(scplugins.BasicTestsOpt) {
-		plugins = append(plugins, basicTestsPlugin)
-	}
-	if viper.GetBool(scplugins.OLMTestsOpt) {
-		plugins = append(plugins, olmTestsPlugin)
-	}
-	// find external plugins
-	pluginDir := viper.GetString(PluginDirOpt)
-	if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
-		log.Warnf("Plugin directory not found; skipping external plugins: %v", err)
-		return plugins
-	}
-	if err := os.Chdir(pluginDir); err != nil {
-		log.Warnf("Failed to chdir into scorecard plugin directory: %v", err)
-		return plugins
-	}
-	files, err := ioutil.ReadDir("bin")
-	if err != nil {
-		log.Errorf("Failed to list files in %s/bin; skipping external plugin tests: %v", pluginDir, err)
-		return plugins
+type pluginConfig struct {
+	Name     string                             `mapstructure:"name"`
```

Comment:
Should we remove `Name` based on our other conversation [here](https://github.com/operator-framework/operator-sdk/pull/1641#discussion_r312595491) or is there still a need for a user-configurable name?

Reply:
The only other place we were using it was in `validateConfig`. I've updated the function to indicate the invalid plugin by using it's index in the config array as well as printing the config for the failed plugin to make it easier for users to identify and removed the `Name` field.